src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -37,6 +38,37 @@ public T WithHttpClientFactory(IMsalHttpClientFactory httpClientFactory)
             return (T)this;
         }
 
+        /// <summary>
+        /// Allows developers to configure details about the authorities used, bypassing the need for MSAL
```

Comment:
> bypassing the need for MSAL [](start = 78, length = 28)

I'm assuming this is not what it is. Rather this is about allow customers to set their own valid authorities and thus they have to be careful when crafting this document?